tsl/test/expected/plan_skip_scan-16.out

```diff
@@ -0,0 +1,3979 @@
+                     Index Cond: ((dev > NULL::integer) AND (dev IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_2_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_2_chunk_skip_scan_ht_dev_idx on _hyper_1_2_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev > NULL::integer) AND (dev IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_3_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_3_chunk_skip_scan_ht_dev_idx on _hyper_1_3_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev > NULL::integer) AND (dev IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_4_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_4_chunk_skip_scan_ht_dev_idx on _hyper_1_4_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev > NULL::integer) AND (dev IS NULL))
+                     Heap Fetches: 1
+(19 rows)
+
+:PREFIX SELECT DISTINCT ON (dev_name) dev_name FROM :TABLE WHERE dev_name IS NULL;
+                                                           QUERY PLAN                                                           
+--------------------------------------------------------------------------------------------------------------------------------
+ Unique (actual rows=1 loops=1)
+   ->  Merge Append (actual rows=4 loops=1)
+         Sort Key: _hyper_1_1_chunk.dev_name
+         ->  Custom Scan (SkipScan) on _hyper_1_1_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_1_chunk_skip_scan_ht_dev_name_idx on _hyper_1_1_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev_name > NULL::text) AND (dev_name IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_2_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_2_chunk_skip_scan_ht_dev_name_idx on _hyper_1_2_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev_name > NULL::text) AND (dev_name IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_3_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_3_chunk_skip_scan_ht_dev_name_idx on _hyper_1_3_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev_name > NULL::text) AND (dev_name IS NULL))
+                     Heap Fetches: 1
+         ->  Custom Scan (SkipScan) on _hyper_1_4_chunk (actual rows=1 loops=1)
+               ->  Index Only Scan using _hyper_1_4_chunk_skip_scan_ht_dev_name_idx on _hyper_1_4_chunk (actual rows=1 loops=1)
+                     Index Cond: ((dev_name > NULL::text) AND (dev_name IS NULL))
+                     Heap Fetches: 1
+(19 rows)
+
+-- test constants in ORDER BY
```

Comment:
For some reason, PG 16 does not use the SkipScan node for the following query. Instead, an Index Scan is used. Might be worth checking if this is just a cost-based decision or if the path is not generated.

Reply:
Good catch ... will merge this PR and investigate it